app/file-upload/server/startup/settings.js

```diff
@@ -165,6 +165,20 @@ settings.addGroup('FileUpload', function() {
 				value: 'AmazonS3',
 			},
 		});
+		this.add('FileUpload_S3_SSECustomerAlgorithm', false, {
+			type: 'boolean',
+			enableQuery: {
+				_id: 'FileUpload_Storage_Type',
+				value: 'AmazonS3',
+			},
+		});
+		this.add('FileUpload_S3_SSECustomerKey', false, {
+			type: 'boolean',
```

Comment:
this can be extracted to the same variable since they are the same

Reply:
one is the algorithm selection, the other is the key used to encrypt. they need to be kept separate.